pkg/util/tracing/collector/collector.go

```diff
@@ -91,7 +91,7 @@ func (t *TraceCollector) StartIter(ctx context.Context, traceID uint64) *Iterato
 	tc := &Iterator{ctx: ctx, traceID: traceID, collector: t}
 	tc.liveNodes, tc.iterErr = nodesFromNodeLiveness(ctx, t.nodeliveness)
 	if tc.iterErr != nil {
```

Comment:
Should this method be swallowing this error at all? Is it worthwhile to surface?